desc/compute/_fast_ion.py

```diff
@@ -180,52 +182,49 @@ def fun(pitch_inv):
                     ),
                 )
             )
-            return jnp.sum(v_tau * gamma_c**2, axis=-1)
+            return jnp.sum(v_tau * gamma_c**2, axis=-1).mean(axis=-2)
```

Comment:
the tldr is to replace this mean over the alpha axis with the logic on `gh/Gamma_c`.